roboteam_utils/CMakeLists.txt

```diff
@@ -28,7 +28,7 @@ set(ROBOTEAM_UTILS_SRC
         src/utils/FileLogger.cpp
         src/utils/Teams.cpp
         src/utils/AIData.cpp
-        src/utils/FieldRectangle.cpp
+        src/utils/FastRectangle.cpp
```

Comment:
That would also mean all the other header files should be included in the CMakeLists file. I am not sure if it will in fact change anything at all, but I guess it might be nice? In any case, I consider that not part of this PR.